index.html

```diff
@@ -27,7 +27,164 @@ <h1 id="name">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+    
+  <section class="class-container2">
+    <img src="img/22.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/23.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/24.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/25.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
```

Comment:
**OPTIONAL** - Using proper HTML tags help improve the semantics and optimization SEO of your site. I suggest you use a `button` or `a` tag instead of a `div`. Please try to fix this issue for all project cards as they may apply 👍 👍